libs2eplugins/src/s2e/Plugins/Core/BaseInstructions.cpp

```diff
@@ -797,6 +797,12 @@ void BaseInstructions::handleBuiltInOps(S2EExecutionState *state, uint64_t opcod
             break;
         }
 
+        case BASE_S2E_IS_FORK_ENABLED: { /* s2e_is_forking_enabled */
```

Comment:
IS_FORK and is_forking naming is inconsistent

Reply:
Use BASE_S2E_IS_FORKING_ENABLED for consistency

Reply:
Done